Cucumber/features/project1/step_definitions/users_steps.rb

```diff
@@ -0,0 +1,215 @@
+Given("I launch login page") do
+  @login_page = start_test("https://demo.openmrs.org/openmrs/login.htm")
+end
+
+And(/^I enter username$/) do
+  @login_page.enter_username("admin")
+end
+
+And(/^I enter password$/) do
+  @login_page.enter_password("Admin123")
+end
+
+And(/^I click location$/) do
+  @login_page.click_location
+end
+
+When(/^I click Login button$/) do
+  @login_page.click_login_button
+end
+
+Then(/^I see Home page$/) do
```

Comment:
It is not common to have expect in such cases. We should verify the exact scenario cases and we assume that navigation to pages is working. In this case it's better to write on_page? method and use it like this - 'wait_until(@home_page.on_page?)'